File: Refactor/api/models/transfer.py
class Transfer:
    def __init__(self, reference, transfer_from, transfer_to, transfer_status, created_at, updated_at):
        self.reference = reference
        self.transfer_from = transfer_from
        self.transfer_to = transfer_to
        self.transfer_status = transfer_status
        self.created_at = created_at
        self.updated_at = updated_at

class TransferItem:
    def __init__(self, item_id, amount):
        self.item_id = item_id
        self.amount = amount

File: src/Models/transfer.cs
using System;
using System.Collections.Generic;

namespace ProjectC.Models
{
    public class Transfer
    {
        public required int Id { get; set; }
        public required string Reference { get; set; }
        public required int? TransferFrom { get; set; }
        public required int TransferTo { get; set; }
        public required string TransferStatus { get; set; }
        public DateTime CreatedAt { get; set; } = DateTime.Now;
        public DateTime UpdatedAt { get; set; } = DateTime.Now;
        public required List<ItemInfo> Items { get; set; }
    }
}